cli/batch/service.py

```diff
@@ -0,0 +1,47 @@
+"""
+Provide implementation of the batch.
+"""
+import asyncio
+
+from accessify import implements
+from aiohttp_json_rpc import RpcGenericServerDefinedError
+
+from cli.batch.interfaces import BatchInterface
+
+loop = asyncio.get_event_loop()
+
+
+@implements(BatchInterface)
+class Batch:
+    """
+    Implements batch.
+    """
+
+    def __init__(self, service):
+        """
+        Constructor.
+
+        Arguments:
+            service: object to interact with Remme core API.
+        """
+        self.service = service
+
+    def get(self, batch_id):
+        """
+        Get a batch.
+
+        Arguments:
+            batch_id (string, required): batch identifier.
+        """
+        try:
+            batch = loop.run_until_complete(
+                self.service.blockchain_info.get_batch_by_id(batch_id=batch_id),
+            )
+
+        except RpcGenericServerDefinedError as error:
+            return None, str(error.message)
+
+        except Exception as error:
+            return None, str(error)
+
+        return batch, None
```

Comment:
We do not need to return a dictionary with key `data`, just its context, so `return batch.get('data')`. Remember to also change the documentation example.

Reply:
Done.